### PR TITLE
ADHOC added instance tag propagation to created snapshops, 

### PR DESCRIPTION
**Pull request by aboritskiy**
so that we can track our costs better.

### CODE REVIEW

File: aws-backup.sh [hunk 1]
```diff
@@ -42,17 +42,30 @@ done <<< $instanceDescription
 # get the volume ids
 volumeIds=`echo "$fullDescription" | awk '/vol-*/ {print $3}'`
 
+#get instance tags
+instanceTags=`echo "$fullDescription" | grep TAG`;
```
Comment (whizkid79) [inline aws-backup.sh]: using backticks is deprecated. better to use $(xx)

File: aws-backup.sh [hunk 2]
```diff
@@ -42,17 +42,30 @@ done <<< $instanceDescription
...
     # create a new snapshot
-    ec2-create-snapshot -O $accessKey -W $secretKey -d "$description" --region $region $volumeId
-
+    snapshotData=`ec2-create-snapshot -O $accessKey -W $secretKey -d "$description" --region $region $volumeId`;
+    printf "Created snapshot:\n---------\n$snapshotData\n--------\n";
+
+    #assign instance tags to the snapshot
+    snapshotId=`echo "$snapshotData" | awk '/snap-*/ {print $2}'`;
+    tagsArgs="";
+    while read -r ignore1 ignore2 ignore3 tagName tagValue; do
```
Comment (whizkid79) [inline aws-backup.sh]: Let's have a chat on that one

File: aws-backup.sh (hunk 1 above)
Comment (aboritskiy) [inline aws-backup.sh]: fixed

File: aws-backup.sh (hunk 2 above)
Comment (aboritskiy) [inline aws-backup.sh]: as discussed `-r` helps to read multiline content, and actually http://www.shellcheck.net/ complains when we don't provide `-r` flag.

**Pull request merged**